src/Makefile.am

```diff
@@ -74,7 +74,7 @@ libproj_la_SOURCES = \
 	jniproj.c pj_mutex.c pj_initcache.c pj_apply_vgridshift.c geodesic.c \
 	pj_strtod.c \
 	\
-	pj_observation.c
+	pj_observation.c pj_cart.c
```

Comment:
Keeping with the tradition the file should be ```PJ_cart.c``` since all files with "projections" are named that way, and the rest uses the lower-cases nomenclature. I don't like it either (I'd prefer to organise the files in directories instead).